code/modules/projectiles/projectile.dm

```diff
@@ -288,7 +288,6 @@
 	var/hit_mod = 0
 	switch(target_mob.mob_size)
 		if(120 to INFINITY)
-			hit_mod = -6
```

Comment:
What's up with this one?

Reply:
from what i could tell that was what was generating the random missses. im not really a coder so i may be wrong

Reply:
this one increases the hit chance if a mob is bigger than Iriska (megafauna and mechs)

Reply:
> this one increases the hit chance if a mob is bigger than Iriska (megafauna and mechs)

so a negative hit mod indicates decreased chance to hit. wierd

Reply:
reverted that part